packages/kbn-test/src/kbn_client/kbn_client_import_export.ts

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { inspect } from 'util';
+import Fs from 'fs/promises';
+import Path from 'path';
+
+import FormData from 'form-data';
+import { ToolingLog, isAxiosResponseError, createFailError } from '@kbn/dev-utils';
+
+import { KbnClientRequester, uriencode, ReqOptions } from './kbn_client_requester';
+import { KbnClientSavedObjects } from './kbn_client_saved_objects';
+
+interface ImportApiResponse {
+  success: boolean;
+  [key: string]: unknown;
+}
+
+interface SavedObject {
+  id: string;
+  type: string;
+  [key: string]: unknown;
+}
+
+async function parseArchive(path: string): Promise<SavedObject[]> {
+  return (await Fs.readFile(path, 'utf-8'))
+    .split('\n\n')
+    .filter((line) => !!line)
+    .map((line) => JSON.parse(line));
```

Comment:
Could be `.map(JSON.parse)`.  No biggie really.

Reply:
Yeah, that would probably work but I'm generally opposed to passing methods that aren't intentionally designed as operators to functions like `Array#map()`, which passes three arguments to the function and changes its `this` context.

Reply:
Yeah fair enough, but could be `.map(JSON.parse.bind(JSON))` or `.map(JSON.parse.bind(null))` :)
But whatevz lol